.github/workflows/ci_cd.yml

```diff
@@ -71,28 +71,43 @@ jobs:
       - uses: actions/download-artifact@v3
         with:
           name: Documentation-pdf
+          path: ~/dist
 
-      - name: "Deploy stable documentation"
-        uses: pyansys/actions/doc-deploy-stable@v4
+      - uses: actions/download-artifact@v3
         with:
-          cname: ${{ env.DOCUMENTATION_CNAME }}
-          token: ${{ secrets.GITHUB_TOKEN }}
-          doc-artifact-name: Documentation-html
+          name: Documentation-html
+          path: ~/dist
+
+      - name: "Compressing HTML documentation"
```

Comment:
You shouldn't need to explicitely zip artifacts, if you mark a folder for upload then it should be zipped by default

Reply:
Wouldn't downloading it unzip it then?